docs/rules/jsx-sort-props.md

```diff
@@ -98,6 +98,16 @@ With `reservedFirst: ["key"]`, the following will **not** warn:
 <Hello key={'uuid'} name="John" ref="ref" />
 ```
 
+### `ignoreTags`
+
+An array of element (tag) names to ignore when checking the properties order.
```

Comment:
these aren't tag names, they're custom component names, so the terminology needs tweaking

Reply:
Is it necessary to be a custom component name? The rule is not limited to sort props on the components only. It currently sorts the attributes of "normal" HTML elements, too. If you accept the ignore option, it should be possible to whitelist normal tags, too.